README.rdoc

```diff
@@ -0,0 +1,28 @@
+== README
```

Comment:
README is a great opportunity to introduce people to your project. Consider not using the default README file from the rails app.